### ISSUE TITLE
Cave gen stuff

### ISSUE THREAD

**Issue by MrCompost** [Angry-Pixel/The-Betweenlands#120]
More caves so it's like a honeycomb type structure (yum https://honeyandspice.files.wordpress.com/2011/04/honeycomb-toffee-9.jpg)
Dirt patches
Limestone caves
Pitstone (new block) instead of Betweenstone further down. New set of ores to go with it.
Moss gen (may need changing)
Lichen gen (may need changing)
Hanger gen
Cave Grass gen/block

**Comment by vadis365** [comment]
Waiting on new moss model but I'll implement the code for it and CyberBrick can add the block model after. :)

**Comment by TripleHeadedSheep** [comment]
Just dropping this here for now

Caves:

Formations:
- Surface openings 	-> 	holes or rips
- Tunnels 		-> 	long, tunnel like cave mazes
- Caverns 		-> 	Huge rooms with lots of ores
- Lake Caves		->	Cave systems with lake basins on different niveaus (illustration needed)
- Limestone Caves	->	Stalactites and stalagmites generate
- Underground Swamps	->	Cold and damp underground lakes with a small ecosystem (illustration needed)

Ores:
- Sulfur
- Octine
- Valonite
- Life Crystal
- Middle Gem

(Maybe when we add more ores, crystal formations could be added)

Vegetation:
- Moss
- Lichen
- Hanger (underground variant?)
- Roofshrooms
- Pillarstalks
- Glow worms (well yeah... basically vegetation?)
- Cave Grass
- Shootroot
- Ferns

Other Generation:
- Tar Pools
- Stagnant Water
- Stalactites/stalagmites
- Dirt and mud patches

**Comment by MrCompost** [comment]
Yeah we can focus on new features afterwards.

**Comment by TripleHeadedSheep** [comment]
Agreed, but Cave gen is Cave gen. Sure, more ores and all will come later, that's why I didn't add them. But the formations and the other generation are basically what the caves are made of... You can't exclude that if you say you want cave gen done ;)
Vegetation is another story, that will come later
